portal-api/src/v1/errors/user-disabled.error.js

```diff
@@ -0,0 +1,9 @@
+class UserDisabledError extends Error {
```

Comment:
Can we please add some documentation on the class, appreciate other classes do not have it either.
Happy to be a separate PR if required.